docs/configuring-metabase/caching.md

```diff
@@ -7,121 +7,107 @@ redirect_from:
 
-On [paid plans](https://www.metabase.com/pricing/), you can view statistics about query execution time from your [auditing tools](../usage-and-performance-tools/audit.md).
+You can schedule caching to invalidate:
 
-## Caching settings
+- Hourly
+- Daily
+- Weekly
+- Monthly
 
-You can tell Metabase when and what to cache from **Admin settings** > **Caching**:
+We do not yet support lunar cycles.
 
-- [Minimum query duration](#minimum-query-duration)
-- [Cache time-to-live (TTL) multiplier](#cache-time-to-live-ttl-multiplier)
-- [Max cache entry size](#max-cache-entry-size)
+### Adaptive caching policy
 
-### Minimum query duration
+Use a query’s average execution time to determine how long to cache the query's results.
 
-Metabase uses this number to decide whether a question will be cached or not.
+- **Minimum query duration**: Metabase will cache this question if it has an average query execution time greater than this many seconds.
+- **Multiplier**: Metabase will cache questions with an average query execution time greater than this many seconds.
 
-Choose a duration (in seconds) that will trigger the cache. For example, you'd enter "60" if you want to cache all questions that take longer than 1 minute to load ([on average](#average-query-execution-time)).
+On [Pro](https://www.metabase.com/product/pro) and [Enterprise](https://www.metabase.com/product/enterprise) plans, you can view querying and caching stats in the [Metabase analytics](../usage-and-performance-tools/usage-analytics.md).
 
-### Cache time-to-live (TTL) multiplier
+### Don’t cache results
 
-The TTL multiplier tells Metabase how long to persist a cached question result:
+Always re-run the query to refresh results.
 
-> Cache lifetime per question = TTL multiplier x [average execution time](#average-query-execution-time) per question
+## Set caching policies at different levels
 
-For example, if you enter a TTL multiplier of 10, a question that takes 5 seconds on average will be cached for 50 seconds. A question that takes 10 minutes will be cached for 100 minutes. This way, each question's cache lifetime is proportional to that question's execution time.
+{% include plans-blockquote.html feature="Advanced caching controls" %}
 
-### Max cache entry size
+You can set up caching at different levels, from largest to smallest scope:
 
-To prevent cached results from taking up too much space on your server, you can set the maximum size of the cache (per question) in kilobytes.
+- [Setting a default caching policy](#default-caching-policy)
+- [Database caching policy (specific to each connected database)](#database-caching-policy)*
+- [Model caching](../data-modeling/models.md#model-caching)*
```

Comment:
I'd remove it from here. Model caching is very different and I think we should convey it in the docs